packages/viewer/docs/spec.md

```diff
@@ -112,6 +112,9 @@ The following table contains a specification of each of the properties.
 | lang                       | The language for the alt text.                                                                                                                                                                                                                                                                                                           | Frontend |                                                 | en                                      |
 | dpi                        | Resolution in dots per inch of the generated image. This feature scales the formula with a factor of dpi/96.                                                                                                                                                                                                                             | Frontend | Positive integer                                | 96                                      |
 | zoom                       | The scale of the generated image.                                                                                                                                                                                                                                                                                                        | Frontend | Positive floating point number                  | 1                                       |
+| vieweroffset                       | Number of pixels to render in advance.                                                                                                                                                                                                                                                                                                        | Frontend | Positive floating point number                  | 200                                       |
```

Comment:
I'm missing maybe some information on what would that parameter be of use, because if I'm using the viewer for the first time, with that information I would not understand.

Maybe add something like "We render only the formulas on screen. This parameter defines the ..."